Codeinwp/neve
# Views refactor

### Summary
Move views out of business logic.


### Will affect visual aspect of the product

NO


### Test instructions

- Check out the view templates under /views
- If you add them into a child theme, it should replace current behaviour


Closes #323.
